cirq/circuits/circuit.py

```diff
@@ -27,6 +27,8 @@
                     List, Optional, overload, Sequence, Set, Tuple, Type,
                     TYPE_CHECKING, TypeVar, Union)
 
+import html
+
```

Comment:
Nit: No blank line needed here.

Reply:
Resolved in commit [54699e5](https://github.com/quantumlib/Cirq/pull/3221/commits/54699e5df1bc1c130eae57604ebfdc2dd5d9d07f)